packages/data-transport-layer/src/services/run.ts

```diff
@@ -12,6 +12,12 @@ interface Bcfg {
   bool: (name: string, defaultValue?: boolean) => boolean
 }
 
+enum ethNetwork {
+  mainnet = 'mainnet',
+  kovan = 'kovan',
+  goerli = 'goerli',
+}
```

Comment:
Usually enums of string are unnecessary; you can just use string literals https://stackoverflow.com/questions/49761972/difference-between-string-enums-and-string-literal-types-in-ts

Reply:
@snario trying string literals and it doesn't solve the not assignable from string when reading from `config.str` :(

Reply:
I think we should apply any forceful typing at the lowest level as possible, if it means using `as` in this case so be it, but I still think using string as opposed to enums is preferable